src/dom.ts

```diff
@@ -116,7 +116,7 @@ export function contains(parent: Element, node: Node): boolean {
  */
 
 export let create: CreateFunction = function (
-		tagName: string, kwArgs?: CreateArgs, children?: (Node | string)[]): HTMLElement {
+		tagName: string, kwArgs?: CreateArgs, children?: (Node | string)[]): any {
```

Comment:
Yeah, we talked about this, in theory the whole code should be rewritten a different way instead of a interface being slapped on top of of it, but the juice doesn't feel worth the squeeze, so I am comfortable with this.